Specs/DateTime/German/DateTimeModel.json

```diff
@@ -145,17 +175,17 @@
     ]
   },
   {
-    "Input": "Ich komme dann Freitag zurück",
+    "Input": "Ich komme am Freitag zurück",
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "NotSupported": "dotNet, javascript, python",
+    "NotSupported": "javascript, python, dotNet",
```

Comment:
No need to modify the order. Unfortunately this will break some scripts we use to eval the specs. They should be robust to ordering here, but currently they are not.